internal/cli/hud.go

```diff
@@ -93,8 +95,17 @@ func connectHud(ctx context.Context) error {
 	cli := proto.NewHudClient(conn)
 
 	stream, err := cli.ConnectHud(ctx)
+
 	if err != nil {
-		return err
+		log.Printf("Waiting for `tilt up` to start.")
+
+		for {
+			stream, err = cli.ConnectHud(ctx)
+			if err == nil {
+				break
+			}
+			time.Sleep(time.Second)
```

Comment:
I might do a smaller sleep? right now ctrl+C on the hud can be unresponsive for up to a second, which feels lousy as a user. (either that or trap SIGINT/SIGTERM? but that feels like overkill...)